showcase-servers/content-automation-mcp/showcase_servers/common/test_security_common.py

```
@@ -370,3 +370,190 @@ def health():
+    assert header_names == sorted(header_names)
+
+
+def test_log_payload_no_header_values_in_payload(monkeypatch, caplog):
+    """Header values must not appear anywhere in the log payload."""
+    monkeypatch.setenv("LOG_LEVEL", "INFO")
+
+    app = FastAPI(title="No Values Test")
+    security.configure_observability(app)
+
+    @app.get("/ping")
+    def ping():
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `ping`

(ANN202)